platform/vs/docker-physyncd-vs/start.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+PLATFORM_DIR=/usr/share/sonic/platform
+HWSKU_DIR=/usr/share/sonic/hwsku
+
+
+# Remove stale files if they exist
+rm -f /var/run/rsyslogd.pid
+
+supervisorctl start rsyslogd
```

Comment:
There is no need to do this anymore. We recently added a new package to allow for supervisor to handle dependent startup ordering, and now pass a flag to rsyslogd to prevent the need to delete the PID file. Can you please use the following PR as a guide and apply changes as appropriate?

https://github.com/Azure/sonic-buildimage/pull/4599

Reply:
@jleveque thanks for that catch, good eyes. Basically took what was done by platform/vs/docker/syncd-vs/start.sh in that PR and modified docker-physyncd-vs/start.sh accordingly (and tested a VS bringup to verify we actually still start :-)

Reply:
@jleveque tests passed (after 11 hours no less :-)